modules/24_trade/standard/input/p24_trade_constraints.cs4r

```diff
@@ -0,0 +1,2 @@
+*NB* by hand, no mrremind or similar
```

Comment:
this file needs to be generated by mrremind. Can be part of the source expert_guess if there is no other source for this information